### ISSUE TITLE
extendedTimeOut=0 Instantly closes notifications

### ISSUE THREAD

**Issue by ashuser** [Foxandxss/angular-toastr#131]
I am assuming this is not intended behavior from looking through the source code. If you create a notification that has a duration of anything other than 0 and set extendedTimeOut=0 the notification closes immediately after the mouse leaves the notification. Based on how timeOut behaves on 0 I expected extendedTimeOut to have no effect on the time out duration if it was set to 0. If this is your intended behavior I apologize! 

My reason for opening this issue is that I would like a notification to persist for its entire timeOut duration regardless of mouse movement over the notification. 

Thanks

**Comment by Foxandxss** [comment]
I copied this behavior from the original Toastr.

Yeah, so you can have toasts that gets closed after X secs, but if you hover them, that timeout gets canceled and another one gets started.

If you put both to 0, they won't close.

There is no way of disabling the mouse over. It is a good feature to add.

**Comment by ashuser** [comment]
Alright. I checked out Toastr as well and there is an open issue [there](https://github.com/CodeSeven/toastr/pull/315). It looks like it is awaiting merge conflict resolution. If it gets merged in to Toastr can we expect it to be implemented here as well?

**Comment by Foxandxss** [comment]
No. The development of this library has nothing to do with the original toastr anymore.

Still, it is an interesting feature, but it would create a breaking change if I do it on that way. I want to release a 2.0.0 which will come with BCs, but I am real busy lately ;(
